### ISSUE TITLE
Licenceing

### ISSUE THREAD

**Issue by wobbol** [ErgoDox/tmk_ErgoDox#25]
The project license file says GPLv3, but lots of project specific files have GPLv2 or later. This could use some attention.

tmk_keyboard's readme says this:

``` quote
GPLv2 or later. Some protocol files are under Modified BSD License.

Third party libraries like LUFA, PJRC and V-USB have their own license respectively.
```

Some of the project specific code here is copyrighted by different people

Copyright 2013 Oleg Kostyuk cub.uanic@gmail.com
Copyright 2016 Mark Sikora marknsikora@gmail.com
Copyright (c) 2012, 2013 Ben Blazak benblazak.dev@gmail.com

I would like to change the project license to ether "GPLv2 or later" or "GPLv3", but there are some copyrights keeping that from happening without sending some emails. Idealy, there would be one entity to hold the copyright to contributed works, but that feels wrong without being incorporated or anything. 
To recap, the project licensing is a little lopsided. We can fix it, but if someone dies we will need to find their next-of-kin to change the project license which is not ideal. Probably no one wants or needs a corporation for this project, but it would be the most elegant solution to the people dieing bit.

Thoughts anyone?

**Comment by marknsikora** [comment]
Not much we can do here. The code is owned by so many people relicencing would be a pain. Currently we don't have any licensing issues.
- BSD allows us to do whatever we want basically
- GPL3 can use BSD licensed code but not vice versa
- GPL2 is compatible with GPL3

So our project is GPL3. Since our main license file is GPL3 all code added to the project is GPL3 unless explicitly mentioned otherwise. 

As far as worrying if someone dies, who cares? It's all copyleft so it doesn't matter.

**Comment by wobbol** [comment]
Alright, sounds like this is a non-issue. Thanks.
